src/components/select/migrate-to-10.x.md

```diff
@@ -1,3 +1,11 @@
 ### HTML
 
 Icon from [`carbon-elements`](https://github.com/IBM/carbon-elements) package is now used. Vanilla markup should be migrated to one shown in [carbondesignsystem.com](https://next.carbondesignsystem.com/components/select/code) site. React and other framework variants should reflect the change automatically.
+
+### SCSS
+
+The `data-invalid` attribute has moved to account for the new markup
```

Comment:
Nitpick - This change probably is for HTML?

* Old: `data-invalid` on `<select>`
* New: `data-invalid` on `<div class="bx--select-input__wrapper>`